app/views/posts/index.html.erb

```diff
@@ -1,10 +1,29 @@
-<h1>This page shows a user's profile</h1>
-<ul>It shows
-<li>User's Name</li>
-<li>Number of all user's posts</li>
-<li>All User's posts</li>
-<li>comments on user's posts</li>
-<li>Number of likes on each post</li>
-<li>Number of comments on each post</li>
-
+<ul>
+    <li class="user-card">
+        <div class="img-holder"><%= image_tag @user.photo %></div>
+        <div class="card-details">
+        <h4 class="user-name"><%= @user.name %></h4>
+        <p class="num-of-posts">Number of posts: <%= @user.post_count %></p>
+        </div>
+    </li>
+    <% @user_posts.each do |post| %>
+    <li class="user-card">
+        <a href="<%= user_post_path(@user, post) %>">
+        <div class="card-details">
+        <h6 class="post-title">Post <%= post.id %></h6>
+        <p class="post-brief"><%= post.text %></p>
+        <p class="num-of-posts">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+        </div>
+        </a>
+    </li>
+      <% if !post.comments.empty? %>
+        <div class="comments-card">
+            <% post.comments.each do |comment| %>
+                <% user = User.find(comment.user_id) %>
+                <p class="post-brief"><%= user.name%>: <%= comment.text %></p>
+            <% end %>
+        </div>
+      <% end %>
+    <br/>
```

Comment:
- [ Required ] I think it would be nice if you center the elements on this page so that the margins on the `left` and `right` sides are equal. Currently, the margins on your design are not equal one is bigger than the other. If you look at the project wireframe for this milestone the margins are equal. Please see the screenshots below and rectify this.

## Your design
![Screenshot from 2023-09-15 02-32-44](https://github.com/danielochuba/duo-blog/assets/16392046/358c6b52-c9ad-4286-9ec5-a29fb86a42b5)

## Project requirements
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABTYBUDZUHSBPLCZA6ORXJT6ZIDUYHQ)